wpeverest/everest-forms
# Feature/range slider field

### All Submissions:






### Changes proposed in this Pull Request:
This PR adds a Pro field `Range Slider`. This PR is depended on https://github.com/wpeverest/everest-forms-pro/pull/160

### How to test the changes in this Pull Request:

1. Create a form
2. Add the range slider field
3. Use all the options available and see if everything is working properly

### Types of changes:




### Other information:




### Changelog entry

> Feature - Added a new Pro field `Range Slider`
